spec/unit/shoulda/matchers/active_record/have_encrypt_attributes_matcher_spec.rb

```diff
@@ -0,0 +1,139 @@
+require 'unit_spec_helper'
+
+describe Shoulda::Matchers::ActiveRecord::HaveEncryptAttributeMatcher, type: :model do
```

Comment:
Do you mind adding specs for the negated_matcher failure message?

Reply:
what you mean ? could you give me some examples like other reviews ?

Reply:
What I meant was to use the matcher in the negated version and assert the failure message.
A quick can be found on : https://github.com/thoughtbot/shoulda-matchers/blob/baabf89127d712855684b351073764026a6c1df1/spec/unit/shoulda/matchers/active_model/validate_comparison_of_matcher_spec.rb#L164-L182